test/test_mean_opacities.cpp

```diff
@@ -81,8 +81,9 @@ TEST_CASE("Mean neutrino opacities", "[MeanNeutrinos]") {
     neutrinos::Gray opac_host(kappa);
     neutrinos::Opacity opac = opac_host.GetOnDevice();
 
-    neutrinos::MeanOpacity<pc> mean_opac_host(
+    neutrinos::MeanOpacityCGS mean_opac_host(
```

Comment:
Out of curiosity would the `MeanOpacity<CGS>` or something similar still work? Presumably yes right?